app/components/ITunesCard/tests/index.test.js

```diff
@@ -0,0 +1,108 @@
+/**
+ *
+ * Tests for ITunesCard
+ *
+ */
+
+import React from 'react';
+import { fireEvent } from '@testing-library/dom';
+import { renderWithIntl } from '@utils/testUtils';
+import { translate } from '@app/components/IntlGlobalProvider';
+import ITunesCard from '../index';
+import { mockData } from './mockData';
+
+describe('<ITunesCard />', () => {
+  let track, onActionClick;
+
+  beforeEach(() => {
+    jest.useFakeTimers();
+    track = mockData;
+    onActionClick = jest.fn();
+  });
+  afterEach(() => {
+    jest.restoreAllMocks();
```

Comment:
Why do we need this ?

Reply:
oh, that is a mistake,  forgot to delete that . Thank you for pointing out.
Will update in next commit